pkg/fileutils/download.go

```diff
@@ -41,6 +41,17 @@ func DownloadFile(dest string, f limayaml.File, decompress bool, description str
 	return res.CachePath, nil
 }
 
+// CachedFile returns path in cache.
```

Comment:
> path in cache

Is this relative? abosolute?

Reply:
Changed to use the same Result, with the same (absolute) CachePath as already being used in Download